backend/models/time.py

```diff
@@ -24,70 +27,49 @@ def _retry_if_result_none(result):
     return result is None
 
 
-class OSRMDrivingTime(Measurer):
-    """
-    Class for OSRM driving time measurements.
-
-    Uses the `table` OSRM API endpoint.
-    """
+# TODO - Abstract as ApiMeasurer.
+class ApiDrivingTime(Measurer):
+    """Class for MapBox driving time measurements."""
 
-    def __init__(self, api_url=None, early_exit_outer_radius=120.0 * 10**3):
-        """Initialize the distance class with the OSRM URL."""
-        # FIXME: Once the API accepts distances, enable a `distance` mode.
-        logger.debug('Initializing OSRM measurer.')
-        self.api_url = api_url or os.getenv('OSRM_URL')
+    def __init__(self, api_key=None, early_exit_outer_radius=120.0 * 10**3):
+        """Initialize the distance class with a MapBox API key."""
+        self.api_key = api_key or ''
         self._haversine_measurer = distance.HaversineDistance()
         self.early_exit_outer_radius = early_exit_outer_radius
+        self.max_matrix_size = 25
 
-    @staticmethod
-    def _represent_point_as_str(point):
-        """Represent a point dictionary in the format (longitude,latitude)."""
-        return '{lng},{lat}'.format(lat=point.latitude, lng=point.longitude)
-
-    @retry(retry_on_result=_retry_if_result_none, stop_max_attempt_number=10, wait_fixed=2000)
-    def measure_between_two_points(self, point_a, point_b):
+    @rate_limited(1, 1.0)  # 1 per second - period=1, every=1.0.
+    @retry(wait_fixed=500, stop_max_attempt_number=2)
+    def get_matrix(self, source_points, destination_points):
```

Comment:
Should we make some of these methods private (e.g., the ones that return times in seconds)?